README.md

```diff
@@ -25,42 +24,155 @@ The following platforms are officially supported (tested):
 If you want to know how to build this project and contribute to it, please
 check out the [Contributing Guide](CONTRIBUTING.md).
 
+## Getting Started
 
-## Usage
+To get started, create a directory for your project and follow the steps below to set up your development environment and run the client.
 
-Please also refer to source of the [CLI tool](https://github.com/frequenz-floss/frequenz-client-reporting-python/blob/v0.x.x/src/frequenz/client/reporting/cli/__main__.py)
-for a practical example of how to use the client.
+### 1. Install VS Code
+
+Download and install **Visual Studio Code** from the [official website](https://code.visualstudio.com/).
+
+Once installed, add the following extensions:
+
+- [Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python)  
+- [Jupyter extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter)  
+
+These enable Python development and interactive notebooks within VS Code.
+
+### 2. Set Up a Virtual Environment
+
+It is recommended to use a **virtual environment** for isolating dependencies.
+When using a virtual environment, you create something like a sandbox or containerized workspace that isolates dependencies and packages so they don’t conflict with the system or other projects. After creating the virtual environment, you can install all packages you need for the current project. To set up an initial virtual environment for using the Reporting API, go through the following steps:
+
+#### Linux / macOS
+
+```bash
+### Go to your projects directory, open a terminal and run the following commands
+
+# Create a virtual environment
+python3 -m venv .venv
+
+# Activate the environment
+source .venv/bin/activate # when using bash
+source .venv/bin/activate.fish # when using fish
+
+# Install the version of the reporting client you want to use
+pip install frequenz-client-reporting==0.18.0
+
+# Install python-dotenv
+# --> Used to load environment variables from a `.env` file into the projects environment
+pip install python-dotenv
+
+# Install ipkernel
+# --> Python execution backend for Jupyter
+pip install ipkernel
+
+# Register the virtual environment as a kernel
+python -m ipykernel install --user --name=myenv --display-name "Python (myenv)"
+
+# Install pandas
+pip install pandas
+
+# Deactivate the environment
+deactivate
+```
+
+#### Windows (PowerShell)
```

Comment:
We are not officially supporting Windows yet, so I wouldn't include a section on how to install in a platform we don't officially support. Maybe we should start supporting Windows, but that's a separate discussion. Again, all of this is pretty general, so we probably should include it in docs for all projects.

Reply:
In the context of making this a separate tutorial for newbies, we could even leave this section with a big warning saying that Windows is not YET officially supported.